src/main/java/com/hida/MinterController.java

```diff
@@ -0,0 +1,253 @@
+package com.hida;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.io.IOException;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * A controller class that paths the user to all jsp files in WEB_INF/jsp.
+ *
+ * @author lruffin
+ */
+@Controller
+public class MinterController {
+
+    // fields for minter's default values
+    private final String CONFIG_FILE = "minter_config.properties";
+    private String prependedString;
+    private String prefix;
+    private boolean isSequential;
+    private boolean isAuto;
+    private String idType;
+    private int length;
+    private String charMap;
+
+    /**
+     * Creates a path to mint ids. If parameters aren't given then printPids
+     * will resort to using the default values found in minter_config.properties
+     *
+     * @param input - requested number of ids to mint
+     * @param model - serves as a holder for the model so that attributes can be
+     * added.
+     * @param parameters - parameters given by user to instill variety in ids
+     * @return - paths user to mint.jsp
+     */
+    @RequestMapping(value = {"/mint/{input}"},
+            method = {org.springframework.web.bind.annotation.RequestMethod.GET})
+    public String printPids(@PathVariable String input, ModelMap model,
+            @RequestParam Map<String, String> parameters) {
+
+        // message variable to be sent to mint.jsp
+        String message;
+        try {
+            int amount = Integer.parseInt(input);
+
+            System.out.print("retrieving data...");
+            // retrieve default setting
+            this.retrieveDefaultSetting();
+
+            // choose between default and given client parameters
+            this.setDefaultSetting(parameters);

```

Comment:
We can't do this.  Per our discussing about mutliple concurrent web requests, if after executing this line, a second requests also comes in with differe parameters, the ones we just set will be overwritten